.changeset/odd-files-attend.md

```diff
@@ -0,0 +1,5 @@
+---
+"@wpengine/wp-graphql-content-blocks": patch
```

Comment:
As @jasonbahl noted, this will change the schema in a breaking way. Should we make this a major bump? Thoughts @theodesp @blakewilson?

Reply:
I would argue that this fixes a bug instead of introducing a feature.

Reply:
@theodesp Does fixing the bug cause breaking changes though? I'm thinking about it from a user's perspective: If they see a `patch` release for a bug fix, download the latest, and their environment breaks, a `major` bump may be more appropriate here.

Reply:
>  If they see a patch release for a bug fix, download the latest, and their environment breaks, a major bump may be more appropriate here.

Worse, even if they have major autoupdates disabled or are using semver in Composer, then this break can happen automatically without anyone needing to actively "download the latest".

Reply:
I think we should do the major bump. The break is there regardless. SemVer says this about bug fixes:

> Bug fixes not affecting the API increment the patch version, backward compatible API additions/changes increment the minor version, and backward incompatible API changes increment the major version.